Dockerfile

```diff
@@ -1,11 +1,25 @@
-FROM golang:1.18 as workspace
+# dummy value for linux builds
+ARG BASE_IMAGE=scratch
```

Comment:
We might even remove the default `scratch` value?

Reply:
Done!